ASWF/meetings/tsc/2020-11-02.md

```diff
@@ -0,0 +1,203 @@
+      hierarchy.
+    - Sean: your thought is then that OCIO should only take image to certain 
+      point? If review tool did want dynamic parameters, it handles it on its 
+      own and then hands off to OCIO. Seems like more room for error. Nice to 
+      handle it in OCIO where transform is consistent. I see both sides points, 
+      but final control of pixels is a bit exaggerated. Every DCC has exposure 
+      and gamma slider. No concern that when I tweak those I am tweaking the 
+      config. Imagining an OCIO display with two views, one with the canonical 
+      look, and another called "editable". You would have tw switch between 
+      them, allowing a supe to make creative decisions when needed, but would 
+      require explicit switching between display chains. Don't see any 
+      operation which would require editing the original config.
+    - Mark B: Not about editing the config; about expressing changes made when 
+      editing. 
+    - Sean: CDL-centric workflow for example. Would need to bake the changes 
+      into a file (CTF) which could be referenced. Hard though since you can't 
+      switch between dynamic and file transform. Like that this gives more 
+      capability than what a CDL can, but am safe in knowledge that decisions 
+      can be reproduced.
+    - Mark B: New file format can be designed by ACES to handle additional 
+      changes. Just problem with live aspect.
+    - Sean: Natural extension of DisplayTransform. Causing more issues since 
+      can be placed anywhere in config.
+    - Doug: The dynamic aspect is what seems to be most troubling. See that as 
+      an implementation detail. There's nothing preventing an application from 
+      editing parameters. The dynamic parameter is only there to allow doing it 
+      interactively with GPU. Could bake out new transform continually. Whether 
+      something can be edited is separate from whether developer uses these 
+      tools.
+    - Mark B: A DCC author will implement it at some point.
+    - Doug: Most tools already have CDL transform.
+    - Mark B: But none allow you to edit it live.
+    - Doug: Dynamic feature there as a convenience. There because of exposure 
+      contrast in viewing pipeline. That tag is there to say doing exposure at 
+      front and gamma at back. Doesn't need to be there for grading parameters.
+    - Kevin: Can do dynamic functionality right now with search paths, etc. We 
+      abuse that for CDL selection, an explicit choice.
+    - Doug: Thanks for the feedback. We have response for all of these. In 
+      terms of written response, would you like answers in this doc, or in 
+      separate doc?
+    - J: Separate doc is preferred.
+    - Michael: Will be good to share these topics and conversation with wider 
+      community too, for additional input. We can continue this discussion 
+      after we all have time to think through the concerns raised.
+
```

Comment:
Please add:

    - *Update:* After the meeting, Doug provided a written response to the ILM questions:
<https://docs.google.com/document/d/1HeA9fJvrxAVoUQ3M1p8K3gAgd7cLkHUmChCoPfmyZvQ/edit?usp=sharing>

Reply:
Thanks @doug-walker . Added.